scripts/python/apiview-syncdb/requirements.txt

```diff
@@ -0,0 +1 @@
+azure-cosmos
```

Comment:
I was about to say something about floating recs, but the claim is that track 2 means no breaking changes right?

Reply:
Ah, and we're getting `azure-core` from inheritance from `azure-cosmos.`